sdks/python/container/base_image_requirements_manual.txt

```diff
@@ -32,14 +32,11 @@ cython<1
 # some versions of libraries that launch Beam pipelines, like tensorflow-transform.
 # Leaving 'future' in our containers for now prevent breaking tft users.
 future
-# TODO: Remove the upper bound once Tensorflow 2.11 is released.
-# https://github.com/apache/beam/issues/23355
-google-cloud-profiler<4.0.0
 guppy3  # Memory profiler
 mmh3  # Optimizes execution of some Beam codepaths. TODO: Make it Beam's dependency.
 nltk  # Commonly used for natural language processing.
 nose==1.3.7  # For Dataflow internal testing. TODO: remove this.
 python-snappy  # Optimizes execution of some Beam codepaths.
 scipy
 scikit-learn
-tensorflow
+protobuf
```

Comment:
this is already a beam dependency, no need to mention it here

Reply:
Ack. will remove it in a different commit and add tensorflow once 2.12 is released.